src/js/components/__tests__/__snapshots__/README-test.js.snap

```diff
@@ -2762,7 +2762,7 @@ Defaults to
   }
 \`\`\`
 ",
-  "Chart": "## Chart
+  "Chart": "## undefined
```

Comment:
Probably has the same root cause as the README changes

Reply:
fixed